includes/Dashboard/Templates/Products.php

```diff
@@ -268,13 +268,16 @@ public function handle_product_add() {
             self::$errors = apply_filters( 'dokan_can_add_product', $errors );
 
             if ( ! self::$errors ) {
+                $timestamp      = time();
```

Comment:
change this line with:
```
$timenow = dokan_current_datetime()->setTimezone( new DateTimeZone( 'UTC' ) );
```

Reply:
Also, do we need to think about existing entries? If so, we are including a upgrader in this release.

Reply:
yes vai, there can be a migrator to fix existing DB entries. possible approach: copy values from `post_date` and `post_modified` columns to `post_date_gmt` and `post_modified_gmt` with proper timezone conversion.